examples/fscloud/main.tf

```diff
@@ -0,0 +1,233 @@
+  activity_tracker_provision     = !local.existing_at
+  logdna_tags                    = var.resource_tags
+  sysdig_tags                    = var.resource_tags
+}
+
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+
+##############################################################################
+# Create CBR Zone and Rules
+##############################################################################
+module "cbr_zone" {
+  source           = "terraform-ibm-modules/cbr/ibm//cbr-zone-module"
+  version          = "1.2.0"
+  name             = "${var.prefix}-VPC-network-zone"
+  zone_description = "CBR Network zone containing VPC"
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [{
+    type  = "vpc", # to bind a specific vpc to the zone
+    value = module.vpc.vpc_crn,
+  }]
+}
+
+module "cbr_rules" {
+  source           = "terraform-ibm-modules/cbr/ibm//cbr-rule-module"
+  version          = "1.2.0"
+  rule_description = "${var.prefix} rule for vpc flow log access to cos"
+  enforcement_mode = "enabled"
+  resources = [{
+    attributes = [
+      {
+        name     = "accountId"
+        value    = data.ibm_iam_account_settings.iam_account_settings.account_id
+        operator = "stringEquals"
+      },
+      {
+        name     = "resourceGroupId",
+        value    = module.resource_group.resource_group_id
+        operator = "stringEquals"
+      },
+      {
+        name     = "serviceInstance"
+        value    = module.cos_fscloud.cos_instance_id
+        operator = "stringEquals"
+      },
+      {
+        name     = "serviceName"
+        value    = "cloud-object-storage"
+        operator = "stringEquals"
+      }
+    ],
+  }]
+  rule_contexts = [{
+    attributes = [
+      {
+        "name" : "endpointType",
+        "value" : "private"
+      },
+      {
+        name  = "networkZoneId"
+        value = module.cbr_zone.zone_id
+    }]
+  }]
+}
+
+
+
+##############################################################################
+# Base OCP Cluster
+##############################################################################
+locals {
+  cluster_hpcs_worker_pool_key_id = regex("key:(.*)", var.hpcs_key_crn_worker_pool)[0]
+  cluster_hpcs_cluster_key_id     = regex("key:(.*)", var.hpcs_key_crn_cluster)[0]
+  cluster_vpc_subnets = {
+    default = [
+      for subnet in module.vpc.subnet_zone_list :
+      {
+        id         = subnet.id
+        zone       = subnet.zone
+        cidr_block = subnet.cidr
+      }
+    ]
+  }
+
+  worker_pools = [
+    {
+      subnet_prefix     = "default"
+      pool_name         = "default" # ibm_container_vpc_cluster automatically names standard pool "standard" (See https://github.com/IBM-Cloud/terraform-provider-ibm/issues/2849)
```

Comment:
"standard" -> "default" (I think someone did a find and replace and accidentally updated this in other examples at some point)